docs/ops/sequence/CTCLoss_4.md

```diff
@@ -0,0 +1,106 @@
+## CTCLoss <a name="CTCLoss"></a>
+
+**Versioned name**: *CTCLoss-4*
+
+**Category**: Sequence processing
+
+**Short description**: *CTCLoss* computes the CTC (Connectionist Temporal Classification) Loss.
+
+**Detailed description**:
+
+This operation is similar [Reference](https://www.tensorflow.org/api_docs/python/tf/compat/v1/nn/ctc_loss)
```

Comment:
TF 2.x declares using dense tensors as input that we are closer to. In the meantime, TF 1.x uses sparse tensors. Also the important difference is a default index for blank: TF 1.x - `N-1`; TF 2.x - `0`. In TF 2,x they state about performance advantage in case using `0`. In both cases they have `blank index` parameter but we don't have it and use `N-1` always. 

1. Should we have `blank` index attribute if we can transpose inputs?
2. Also should we have `unique` attribute as in TF 2.x? I propose not to specify for now. Now we don't have such model for infer.

Reply:
1. I think that we should support both cases and thus we need `blank` attribute.
2. I cannot answer for sure for this question. Need deeper analysis.

Reply:
1. I will `blank` optional input. As in TF 1.x its default value will be `N-1`, where `N` - a number of classes including `blank`, since `CTCGreedyDecoder` always uses `N-1` for coding `blank`.
2. `unique` may be useful attribute for low power hardware like VPU, as they state in TF spec the operation can be faster and memory efficient on TPU. It is like used for pseudo-calculation for CTCLoss value. I will add it as attribute with default value = False. If True, it finds unique elements in `labels` for each object and performs matching with potential alignment.

Reply:
Fixed.